.travis.yml

```diff
@@ -15,8 +15,8 @@ notifications:
     on_pull_requests: false
     on_success: never
 
-before_install:
-  - yarn install
```

Comment:
this is unnecessary. Travis will run `yarn install` prior to building by default